src/components/backup-finish-popup.tsx

```diff
@@ -0,0 +1,58 @@
+import React, {useEffect} from 'react';
+import {Dimensions, StyleSheet} from 'react-native';
+
+import {HapticEffects, vibrate} from '@app/services/haptic';
+
+import {
+  Button,
+  ButtonVariant,
+  LottieWrap,
+  PopupContainer,
+  Spacer,
+  Text,
+} from '@app/components/ui';
+import { I18N } from '@app/i18n';
+import { createTheme } from '@app/helpers';
+
+const animationSize = Dimensions.get('window').width - 116;
+
+interface BackUpFinishPopUp {
+  onSubmit?: () => void;
```

Comment:
non optional field and use `type` instead `interface` pls

Reply:
done